sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/OpenCoverReportParserTest.java

```diff
@@ -123,12 +123,12 @@ public void branchCoverage() throws Exception {
         Assertions.entry(7, 1),
         Assertions.entry(14, 1));
 
-    assertThat(coverage.getLinesBranchCoverage(filePath).size()).isEqualTo(1);
+    assertThat(coverage.getBranchCoverage(filePath).size()).isEqualTo(1);
```

Comment:
as discussed offline, I suggest to add a more complex test with more than 1 branch coverage entry

Reply:
I've added a few more cases and with this I've found and fixed a corner case when multiple branch points are present at the same line